index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>javascriptの練習</title>
+  <script type="text/javascript" src="task.js"></script>
+</head>
+<body>
+  <h1>ToDoリスト</h1>
+  <input type="radio" value="1" checked="checked">すべて
+  <input type="radio" value="2">作業中
+  <input type="radio" value="3">完了
+
+  <table id='targetTable'>
+    <tr>
+      <th>ID</th>
+      <th>コメント</th>
+      <th>状態</th>
+      <th>　　</th>
+    </tr> 
```

Comment:
`thead` と `tbody` を使った方がより良いと思います。

```suggestion
    <thead>
      <tr>
        <th>ID</th>
        <th>コメント</th>
        <th>状態</th>
        <th>　　</th>
      </tr>
    </thead>
    <tbody>
    </tbody>
```

Reply:
thead と tbodyを使用しました。tbodyにアペンドするようにしました。